exporter/opentelemetry-exporter-otlp-proto-http/setup.cfg

```diff
@@ -0,0 +1,57 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[metadata]
+name = opentelemetry-exporter-otlp-proto-http
+description = OpenTelemetry Collector Protobuf over HTTP Exporter
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = OpenTelemetry Authors
+author_email = cncf-opentelemetry-contributors@lists.cncf.io
+url = https://github.com/open-telemetry/opentelemetry-python/tree/main/exporter/opentelemetry-exporter-otlp-proto-http
+platforms = any
+license = Apache-2.0
+classifiers =
+    Development Status :: 5 - Production/Stable
```

Comment:
Might have to have a discussion on whether or not we can mark this as stable since [collector](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.29.0) is technically not stable (even though otlp grpc is stable). Will talk about it in the SIG.

Reply:
We didn't get a chance to talk about this in the SIG but I agree w/ @lzchen, we should mark hold off marking this as stable until the port is confirmed.